### ISSUE TITLE
Package install fails when running "pip install git+https://github.com/nathanblumenfeld/collegebaseball" on Visual Studio

### ISSUE THREAD

**Issue by armstjc** [nathanblumenfeld/collegebaseball#1]
Code output when run:
```
----- Installing 'git+https://github.com/nathanblumenfeld/collegebaseball' -----
Collecting git+https://github.com/nathanblumenfeld/collegebaseball
  Cloning https://github.com/nathanblumenfeld/collegebaseball to c:\users\xme10\appdata\local\temp\pip-req-build-p_5ph3sy
  Running command git clone --filter=blob:none --quiet https://github.com/nathanblumenfeld/collegebaseball 'C:\Users\xme10\AppData\Local\Temp\pip-req-build-p_5ph3sy'
  Resolved https://github.com/nathanblumenfeld/collegebaseball to commit 76746c4ceed6cde9406cbcc6e637df7b0e35147e
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: pandas in c:\program files (x86)\microsoft visual studio\shared\python39_64\lib\site-packages (from collegebaseball==1.0.0a0) (1.4.1)
Requirement already satisfied: numpy in c:\program files (x86)\microsoft visual studio\shared\python39_64\lib\site-packages (from collegebaseball==1.0.0a0) (1.22.3)
Requirement already satisfied: requests in c:\program files (x86)\microsoft visual studio\shared\python39_64\lib\site-packages (from collegebaseball==1.0.0a0) (2.27.1)
Requirement already satisfied: lxml in c:\program files (x86)\microsoft visual studio\shared\python39_64\lib\site-packages (from collegebaseball==1.0.0a0) (4.8.0)
Collecting bs4
  Using cached bs4-0.0.1-py3-none-any.whl
Collecting importlib
  Using cached importlib-1.0.4.zip (7.1 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
  error: subprocess-exited-with-error
  
  python setup.py egg_info did not run successfully.
  exit code: 1
  
  [1 lines of output]
  ERROR: Can not execute `setup.py` since setuptools is not available in the build environment.
  [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed
Encountered error while generating package metadata.
See above for output.
note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
----- Failed to install 'git+https://github.com/nathanblumenfeld/collegebaseball' -----
```

**Comment by armstjc** [comment]
![image](https://user-images.githubusercontent.com/70788681/161597571-e11e663d-e26f-49ff-be1b-50dfad15d436.png)
setuptools is available in the build environment.

**Comment by nathanblumenfeld** [comment]
added a pyproject toml to resolve potential dependency issues related to setuptools

**Comment by nathanblumenfeld** [comment]
changes included in v1.1.0-alpha release
